app/views/docs/storage.phtml

```diff
@@ -0,0 +1,83 @@
+
+<h2>Buckets</h2>
+
+<p> Storage buckets are way to group files into a meaningful collection and give them context. Buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket. Whether or not to encrypt the files, scan with antivirus and more. Let us look at how we can create a bucket and configure it to our needs.</p>
+
+<h3>Create Bucket</h3>
+
+<p>You can create your bucket from your Appwrite project's dashboard or using any of our server side SDKs or directly using our REST API authenticated using  an API Key. In order to create a bucket from dashboard, access your Storage from your project's left navigation panel. Click <b>Add Bucket</b> button and choose your bucket's name. For convenience, you can also set a custom ID for your bucket instead of auto-generated ID. This will create a new bucket and take you to it's settings page, where you can configure various options for your bucket.</p>
+
+<p>In order to create bucket programmatically, you can either use any of our server side SDKs or REST API paired with an API Key. Let's look at a code sample using CURL, how we can create a bucket using our REST API.</p>
+
+<h4>Understanding Bucket Permissions</h4>
+
+As buckets are very similar to Collections we have in database, here again we have similar permission model. Buckets support either bucket level permission or the file level permission to allow different ways to manage access to your buckets and files. Check out the <a href="/docs/permissions">Permissions</a> documentation to learn more about permissions types available.
+
+<h4>Bucket Level Permissions</h4>
+
+<p>With bucket level permission, you only need to provide permissions in your bucket and not in files. If you set the permission to bucket level, file permissions are optional and ignored even if you have set them. This is useful when you want to provide same permission for a collections of files but don't want to repeat the same permission for each file. You provide it once for bucket, set the bucket permission to be bucket level and you are done. If you want to update the permission, you only need to do it once in the bucket. With this permission model, if a user has a read and write permission, they will be able to read all the files and modify all the files in that bucket.</p>
+
+<p>However, with this permission you don't have a granular control over the files inside the bucket. This is more suitable for a use case where you have many people who all have access to same group of files. So instead of adding their permission to each file, you can just create a bucket with bucket level permission and provide them access. Then add all the files they have access to inside that bucket.</p>
```

Comment:
```suggestion
<p>However, with this permission model, you don't have granular control over the files inside the bucket. This model is more suitable for a use case where you have many people who all have access to the same group of files. Instead of adding their permission to each file, you can just create a bucket with bucket level permission and provide them access. Now you can add all the files they should have access to inside that bucket.</p>
```